.git-cz.json

```diff
@@ -1,4 +1,5 @@
 {
   "disableEmoji": false,
-  "format": "{type}{scope}: {emoji}{subject}"
+  "format": "{type}{scope}: {emoji}{subject}",
+  "gitmoji": false
```

Comment:
Instead of hand-cuffing to `gitmoji` later in the review it is suggested to move to `theme`.

- `"theme": "default"`
- `"theme": "gitmoji"`